packages/flutter/lib/src/services/keyboard_maps.dart

```diff
@@ -0,0 +1,844 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// DO NOT EDIT -- DO NOT EDIT -- DO NOT EDIT
+// This file is generated by dev/tools/gen_keycodes/bin/gen_keycodes.dart and
+// should not be edited directly.
+//
+// Edit the template dev/tools/gen_keycodes/data/keyboard_maps.tmpl instead.
+// See dev/tools/gen_keycodes/README.md for more information.
+
+import 'keyboard_key.dart';
+
+/// Maps Android-specific key codes to the matching [LogicalKeyboardKey].
```

Comment:
Wondering if it would make sense to link from the dart docs of all these generated maps here and in the previous file to https://github.com/flutter/flutter/blob/master/dev/tools/gen_keycodes/README.md. That way, people who look at this thing on docs.flutter.io have a chance of finding out where the "magic"numbers in this map are coming from.

Reply:
Well, except that I really don't want them to try and unpack the magic numbers: they're supposed to be fully opaque.  The only time you should need to understand that is if you're porting Flutter to a new platform.

Reply:
Sounds good.